2019-02-24 18:47:10.684 [main] ERROR org.shgbit.log4j2.log4j2Test.main() Line: 26 - Did it again!
